dev/conf/default.conf

```diff
@@ -0,0 +1,20 @@
+server {
+    listen              443 ssl;
```

Comment:
didn't you say it listens on 8443? what am i missing?

Reply:
Inside the container it listens to 443.
The docker-compose redirects 8443 from outside to 443 inside, so everything is good 👍